app/ts-meta/meta/raft_wrapper.go

```diff
@@ -248,3 +248,10 @@ func (l *raftLayer) Dial(address raft.ServerAddress, timeout time.Duration) (net
 func (l *raftLayer) Accept() (net.Conn, error) { return l.ln.Accept() }
 
 func (l *raftLayer) Close() error { return l.ln.Close() }
+
+func (r *raftWrapper) LeadershipTransfer() error {
+	if !r.IsLeader() {
```

Comment:
It is not necessary to judge whether it is a leader, because LeadershipTransfer can only execute on leader successfully.